crates/sui-open-rpc/samples/objects.json

```diff
@@ -9,22 +9,22 @@
         "fields": {
           "description": "An NFT created by the Sui Command Line Tool",
           "id": {
-            "id": "0x0bd2ba409b00487652fe84aa1ffe61cb1623062f",
+            "id": "0xc3e9d83dcae448e0dd11e4b6f335ff5c799e8634",
```

Comment:
curious why do we want to change these IDs and addresses?

Reply:
are they auto generated?

Reply:
Yes, they are generated through https://github.com/MystenLabs/fastnft/blob/e7778f4af425a909c4f453eb2c5104ccead64f54/crates/generate-json-rpc-spec/src/main.rs#L79

Reply:
@666lcz sry what I meant was, if I want to generate this, how do I do that? thanks!

Reply:
You can run `cargo -q run --example generate-json-rpc-spec -- record` after https://github.com/MystenLabs/sui/pull/3103 is merged. You can also just hit the run button in VSCode when you open the file
![CleanShot 2022-07-08 at 10 21 20](https://user-images.githubusercontent.com/76067158/178040355-b0759d25-3826-49c2-84e8-20ce0d59fed2.png)